website/content/docs/connect/gateways/mesh-gateway/peering-via-mesh-gateways.mdx

```diff
@@ -7,7 +7,7 @@ description: >-
 
 # Enabling Peering Control Plane Traffic
 
-In addition to [service-to-service traffic routing](/consul/docs/connect/gateways/mesh-gateway/service-to-service-traffic-peers),
+In addition to [service-to-service traffic routing](/consul/docs/connect/gateways/mesh-gateway/service-to-service-traffic-partitions),
```

Comment:
```suggestion
This topic describes how to configure a mesh gateway to route control plane traffic between Consul clusters that share a peer connection. For information about routing service traffic between cluster peers through a mesh gateway, refer to [Enabling Service-to-service Traffic Across Admin Partitions](/consul/docs/connect/gateways/mesh-gateway/service-to-service-traffic-partitions).
```
I know that it's not in the scope of this PR, but we should take this opportunity while editing this page to add some clarity about the what this page is about.

Reply:
Thanks a lot for the suggestion, I've extended the section accordingly.